articles/portal/ptl-ref-portal-api.md

```diff
@@ -3678,6 +3678,119 @@ Returns a CSV report for the whole month specified including the date provided.
   Consumption (GB) | Average storage used over the month in gigabytes
   Cost | Amount that would be charged if this were for a whole month (in pounds)
 
+## GET /api/billing/billing-csv
+
+Returns a CSV containing billing data for the period given.
```

Comment:
Is it worth indicating this data only covers VMware, OpenStack and DR to the Cloud?

Reply:
Do any other services have org ids that could be used? If there are I'll add a note in

Reply:
Sorry don't know